corehq/apps/app_manager/tests/test_app_manager.py

```diff
@@ -54,8 +54,7 @@ def setUpClass(cls):
         cls.domain = 'test-domain'
         create_domain(cls.domain)
 
-        with codecs.open(os.path.join(os.path.dirname(__file__), "data", "very_simple_form.xml"), encoding='utf-8') as f:
-            cls.xform_str = f.read()
+        cls.xform_str = cls.get_xml('very_simple_form').decode('utf-8')
```

Comment:
any idea whether `decode('utf-8')` should be part of `get_xml`?

Reply:
I don't know. It looks like about 1/4 of the time get_xml is called it's immediately decoded (70 out of 369 instances). The handful of calls that aren't in tests, which include form processor, almost all *don't* use decode.